training/src/anemoi/training/train/train.py

```diff
@@ -284,7 +286,7 @@ def tensorboard_logger(self) -> pl.loggers.TensorBoardLogger:
         return get_tensorboard_logger(self.config)
 
     @cached_property
-    def last_checkpoint(self) -> str | None:
+    def last_checkpoint(self) -> Path | None:
```

Comment:
is this a change on the type hint, or a correction? (ie it was a already a Path?)